lib/git-manager.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
    getOpenIssues: function(repo) {
       const searchParams = params({}, repo);
       debug("Getting open issues");
-      return github.issues.getAll(searchParams)
+      return github.issues.getForRepo(searchParams)
```

Comment:
We might actually want to use [`listForRepo`](https://octokit.github.io/rest.js/#api-Issues-listForRepo), because it doesn't seem like `getForRepo` is documented for issues in those API docs.

Reply:
This is the right name to use for v15.x: https://github.com/octokit/rest.js/blob/15.x/lib/routes.json#L3768-L3832

Looks like they changed it in v16 and moved everything to its own file / repo: https://github.com/octokit/routes/blob/master/routes/api.github.com/issues/list-for-repo.json

Reply:
Do I misunderstand? It seems like [this](https://github.com/octokit/rest.js/blob/15.x/lib/routes.json#L3379-L3382) is saying `getForRepo` is deprecated in v15.x?

Reply:
Wow, you're right! But the version in our package.lock.json is 15.4.1 which *didn't* have that. I made this change by looking at the installed version in `node_modules`. We could bump the version requirement, but I'd rather not right now.

Reply:
Works for me. Thanks for checking!